.github/workflows/dispatch-build-bottle.yml

```diff
@@ -136,13 +136,19 @@ jobs:
       - name: Setup git
         uses: Homebrew/actions/git-user-config@master
 
+      - name: Set up commit signing
+        uses: Homebrew/actions/setup-commit-signing@master
+        with:
+          signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
+
       - name: Upload and publish bottles on Bintray
         env:
           HOMEBREW_BINTRAY_USER: brewtestbot
           HOMEBREW_BINTRAY_KEY: ${{secrets.HOMEBREW_BINTRAY_KEY}}
+          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
         run: |
           cd ~/bottles
-          brew pr-upload --verbose --keep-old
+          brew pr-upload --verbose --keep-old --committer='BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>'
```

Comment:
We have a BrewTestBot email we use elsewhere: https://github.com/Homebrew/homebrew-test-bot/blob/2795d53426986ab47a0154d487d5e518f99fd4f6/lib/test_bot.rb#L122

Reply:
Right, for some reason I thought that was private. I'll change it 👍🏼

Reply:
It might be worth setting it as public on the GitHub account, for consistency reasons. I do see the GitHub noreply email is being used for sign-offs because the email isn't public.

Reply:
I think it's worth this being generally private rather than public. I think it's fine as-is and we can update the above case to use the private link when possible.

Reply:
I'm fine whatever, but it should be made consistent throughout rather than bits and bobs using one or the other.